<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVLMGrib_pc_ui</class>
 <widget class="QDialog" name="DialogVLMGrib_pc_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VLM grib download</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>234</width>
        <height>168</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radio1">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio2">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio3">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio4">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio5">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_download">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogVLMGrib_pc_ui</receiver>
   <slot>slot_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_download</sender>
   <signal>clicked()</signal>
   <receiver>DialogVLMGrib_pc_ui</receiver>
   <slot>slot_download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_download()</slot>
  <slot>slot_cancel()</slot>
 </slots>
</ui>
